migration-from-4x.md

```diff
@@ -0,0 +1,328 @@
+
+## Editor-Core
+
+### Initialization
+
+The initialization process of TinyMCE 5.0 is the same as TinyMCE 4.x. The bootstrap process and initialization events all remain the same.  The main differences are in the `init` configuration, specifically the configuration items for UI components. It still retains a familiar JSON structure. However, many of the configuration options have been simplified.
+
+### Settings
+
+In TinyMCE 5.0, some configurations have been removed because they are no longer necessary or an improved solution has been introduced.
+
+#### Changed Settings
+
+| **Old Settings** |  **Description**|
+| ---------------- | --------------- |
+| Settings for inline styles | Uses CSS stylesheets for custom |
+
+TODO
+
+#### New Settings
+
+TODO
+
+#### Removed Settings
+
+* **fixed_toolbar_container**: Owing to the enhancements to the new inline toolbar behaviour, `fixed_toolbar_container` is not required in TinyMCE 5.0.
+
+TODO
+
+### Methods
+
+* All TinyMCE 4.x methods for creating creating UI components have been removed. New methods have been added for TinyMCE 5.0. For more information, see the [docs]({{site.baseurl}}/components/).
+* No core editor methods were removed (tinymce, editor, selection, on(), etc remain the same).
+
+## Themes
+
+In TinyMCE 5.0, some themes have been removed and are now combined in a new single responsive theme called **Silver**.
+
+| **Removed Theme** | **Replaced by** |
+| ----------------- | --------------  |
+| [Modern](https://www.tiny.cloud/docs/themes/modern/) | Silver |
+| [Modern inline](https://www.tiny.cloud/docs/general-configuration-guide/use-tinymce-inline/) | Silver Inline |
+| [Inlite(Distraction-free Editor)](https://www.tiny.cloud/docs/themes/inlite/) | Silver (distraction free configuration) |
+| [Mobile](https://www.tiny.cloud/docs/themes/mobile/) | Silver (responsive to screen size)  |
+
+### Removed Themes
+
+#### Inlite
+
+The Inlite theme is no longer supported in TinyMCE 5.0. The features that the Inlite theme used to provide are now available as a plugin. For a workaround, use the following configuration:
+```
+{
+  theme: 'silver',
+  inline: true,
+  toolbar: false,
+  plugins: [ 'inlite' ]
+}
+ ```
+This will provide a similar but improved distraction free experience in TinyMCE 5.0.
+
+#### Modern
+
+The Modern theme is no longer supported in TinyMCE 5.0.  The modern themes Ui library `tinymce.ui.*` has also been deleted. This change may impact integrations depending upon the level of customization.
+
+For changes required, refer to the following table of examples:
+
+| Customization Level | Description | Impact |
+| ------------------- | ----------- | ------ |
+| Minor | Some custom buttons | no Ui fixes required, update `addButton` configuration to TinyMCE 5.0 format |
```

Comment:
Capitalise "no" and add a link to custom components docs?

Reply:
Done